### ISSUE TITLE
convenient macro for directed composition?

### ISSUE THREAD

**Issue by slwu89** [AlgebraicJulia/AlgebraicDynamics.jl#56]
Hi, is there a similar macro to `@relation` that can be used to define the composition pattern for systems of continuous machines? The examples construct them by hand for directed composition. Thanks!

**Comment by epatters** [comment]
Yes, you can use the `@program` macro to define a directed wiring diagram using the ordinary notation for function application. I don't know if there are examples of usage with AlgebraicDynamics (@slibkind?) but you can see some examples of using the macro [here](https://github.com/AlgebraicJulia/Catlab.jl/blob/master/test/programs/ParseJuliaPrograms.jl).

**Comment by slibkind** [comment]
Unfortunately, there aren't (as of yet) any examples using the `@program` macro in AlgebraicDynamics
